pipeline/metadata/satellite.py

```diff
@@ -341,8 +341,7 @@ def _key_by_date_domain(row: Row) -> Tuple[DateDomainKey, Row]:
 def _partition_test_and_control(row_tuple: Tuple[DateDomainKey, Row],
```

Comment:
The original intent of this function was to separate measurements to control resolvers and measurements to test resolvers. (`row['is_control_ip']`)

Reply:
I would maybe change the name to `_partition_test_and_control_resolvers`

Reply:
I'm now partitioning this into three pieces (domain_controls, ip_controls, and tests) and added a doc comment to make it clearer what the intent is.